tests/State/HubTest.php

```diff
@@ -7,25 +7,19 @@
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 use Sentry\Breadcrumb;
-use Sentry\ClientBuilder;
 use Sentry\ClientInterface;
+use Sentry\Event;
+use Sentry\Integration\IntegrationInterface;
+use Sentry\Options;
 use Sentry\Severity;
 use Sentry\State\Hub;
-use Sentry\State\HubInterface;
 use Sentry\State\Scope;
 
 final class HubTest extends TestCase
 {
-    public function testConstructorCreatesScopeAutomatically(): void
-    {
-        $hub = new Hub(null, null);
-
-        $this->assertNotNull($this->getScope($hub));
-    }
-
     public function testGetClient(): void
     {
-        /** @var ClientInterface|MockObject $client */
+        /** @var ClientInterface&MockObject $client */
```

Comment:
Isn't this PHPDoc superfluous with the `phpstan-phpunit` extension?

Reply:
Yes it is, however it helps the autocompletion of the IDE that otherwise would not recognize that the type of the variable is a `ClientInterface` and not only a `MockObjec`. I think that this annotation is still wrong as it should probably be `MockObject<ClientInterface>`, but then it's useless again as PHPStorm (in my case) doesn't recognize the syntax for objects that are not `iterable`

Reply:
If that's the case, you should use the PHPUnit plugin for PHPStorm and drop the annotation: https://github.com/Haehnchen/idea-php-phpunit-plugin

(it's a fork of the previous one which was broken)